Installation error when running Java 1.9

On macOS, installing ODK Aggregate throws a NoClassDefFoundError error.

https://forum.opendatakit.org/t/odk-aggregate-installation-error-on-mac-sierra/10232
https://forum.opendatakit.org/t/odk-aggregate-installer-failure-on-macos/9978

I am still getting this problem. To reproduce, I did the following..

1. Put `org.gradle.java.home=/Library/Java/JavaVirtualMachines/jdk1.8.0_152.jdk/Contents/Home` in my properties so gradle uses Java 8 to compile. It can't compile with Java 9.
1. Run `./gradlew clean build -x test -PwarMode=installer && ./gradlew buildInstaller`
1. Use InstallBuilder to generate an installer.
1. Run the installer and have it launch the uploader.

<img width="504" alt="screen shot 2017-12-08 at 15 08 05" src="https://user-images.githubusercontent.com/32369/33788870-9c6e45fc-dc29-11e7-9858-2bca7f0da63f.png">

```
$ java -jar ODKAggregateAppEngineUpdater.jar
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/commons/cli/ParseException
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:375)
	at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:56)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.cli.ParseException
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:466)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:563)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
	... 3 more
```
```
$ java --version
java 9.0.1
Java(TM) SE Runtime Environment (build 9.0.1+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.1+11, mixed mode)
Veritas:~/Desktop/ODK Aggregate >
```

--- a/installer/gae.zip
+++ b/installer/gae.zip
@@ -1,3 +1,3 @@
 version https://git-lfs.github.com/spec/v1
-oid sha256:abb0f5c1b2ebf1377ddde071f67f8491a2ed44a1d883acb55d7158ebd98e8a6e
-size 1434856
+oid sha256:e2c28bbc689c4b888e4780e4ea8b5ebe975f2829b9bfdcd8041e4594c0238e05
+size 1396896

